### ISSUE TITLE
Premade tutorial project

### ISSUE THREAD

**Issue by nightblade9** [db0/godot-card-game-framework#172]
I looked through the installation instructions. A lot of them mention copying things from the core folder into a new folder. It looks like a lot of steps.

Is there any way you can create a premade project that you can just clone/unzip and go? It sounds like you can include any older version of the framework, and users can upgrade by getting the latest version and copying over core.

**Comment by db0** [comment]
I've actually had an idea of making a script file which would do the quick-start steps automatically for someone who wants to not bother, but the point of the quickstart is to also introduce some concepts to new players, so that they understand the components. Technically one can simply use the custom folder directly without any copying.

**Comment by db0** [comment]
>I actually thought the right approach is to extend (not copy) core classes, so that you can upgrade them later and get the benefits of upgrading (does that work if you copy them?)

That is correct. You should always be extending classes or instancing scenes from the core/ folder, if you want the framework to remain upgradable.

For a quick prototype, keeping the custom/ folder as is and adjusting it from there should work just fine.
